src/AddExperience.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+
+const AddExperience = ({ onCancel }) => {
+  const [title, setTitle] = useState("");
+  const [company, setCompany] = useState("");
+  const [startDate, setStartDate] = useState("");
+  const [endDate, setEndDate] = useState("");
+  const [description, setDescription] = useState("");
+  const [logo, setLogo] = useState("");
+  const [message, setMessage] = useState("");
+
+  const handleSubmit = async () => {
+    const experience = {
+      title,
+      company,
+      start_date: startDate,
+      end_date: endDate,
+      description,
+      logo,
+    };
+
+    try {
+      const response = await fetch("/resume/experience", {
```

Comment:
this is making POST to localhost:3000

our backend is localhost:5000

![image](https://github.com/user-attachments/assets/1657e9d1-1084-4dcd-948c-3bfb2908e443)